linode/manager
# refactor: [M3-6418] - MUI v5 Migration - Components > Toggle

## Description 📝

- I don't even know if this counts as a react tss migration but a ticket existed for the `<Toggle />` but there are no styles applied to it so I just did a clean up and modernization of the component

## How to test 🧪

- Find places in the app that use toggles, verify the toggles still look normal and function normally
